pytest_fixtures/core/broker.py

```diff
@@ -211,3 +211,11 @@ def module_lb_capsule(retry_limit=3, delay=300, **broker_args):
 
     _ = [cap.teardown() for cap in cap_hosts.out]
     Broker(hosts=cap_hosts.out).checkin()
+
+
+@pytest.fixture(scope='module')
+def module_capsule_configured_async_ssh(module_capsule_configured):
+    """Configure the capsule instance with the satellite from settings.server.hostname,
+    enable MQTT broker"""
+    module_capsule_configured.set_rex_script_mode_provider('ssh-async')
```

Comment:
@pondrejk the `mode_provider` update will affect other tests ? as this uses the `module_capsule_configured` fixture which is used across the project. Do you think is it good to reverse the setting to use the `pull-mqtt`  ?

Reply:
@omkarkhatavkar hm, so there is `module_capsule_configured_mqtt(module_capsule_configured)` that uses `set_rex_script_mode_provider('pull-mqtt')` and `module_capsule_configured_async_ssh(module_capsule_configured)` that uses  `set_rex_script_mode_provider('ssh-async')`, both based on module_capsule_configured, is that a concern?